bedrock/firefox/views.py

```diff
@@ -573,6 +573,10 @@ def download_thanks(request):
             template = 'firefox/new/portland/scene2-safe.html'
         elif experience == 'betterbrowser':
             template = 'firefox/new/better-browser/scene2.html'
+        elif switch('firefox_download_scene2_email_experiment') and variant == 'a':
```

Comment:
Maybe I'm a little confused on the plan, but I thought we wanted to roll this new page out as the default because it was the "right thing to do"? I had imagined the A/B copy test would be separate to doing that - but I'm guessing I didn't understand quite right.

Reply:
I'll get clarification on the test. I was under the impression this was going to be temporary and would need to be switch-offable, though I don't know what we're measuring against so maybe it really is the new permanent default (for en-US).